From 92091bde28128caff5621f09ffe52c9f8b0b55f4 Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Wed, 6 Mar 2019 07:57:01 -0500
Subject: [PATCH 1320/1936] Upgrade travis-CI distro version

Change distro from default, Ubuntu trusty, to the newer Ubuntu xenial.
For more information on other configuration changes which could be
used, see:
https://docs.travis-ci.com/user/reference/overview/
---
 .travis.yml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/.travis.yml b/.travis.yml
index 0428a297..4f4a5175 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -3,6 +3,8 @@ language: cpp
 compiler:
     - gcc
 
+dist: xenial
+
 before_install:
     - sudo apt-get install zlib1g-dev libmxml-dev libfftw3-dev dssi-dev libfltk1.3-dev libxpm-dev
     - sudo apt-get install --force-yes cxxtest
-- 
2.47.0

